specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/codeVersion.json

```diff
@@ -0,0 +1,400 @@
+            "description": "Success"
+          },
+          "204": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "CodeVersion"
+        ],
+        "description": "Get code.",
+        "operationId": "CodeVersions_Get",
+        "x-ms-examples": {
+          "Get CodeVersion": {
+            "$ref": "./examples/getCodeVersion.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CodeVersionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CodeVersionVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CodeVersionResource"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "CodeVersion"
+        ],
+        "description": "Create or update code.",
+        "operationId": "CodeVersions_CreateOrUpdate",
+        "x-ms-examples": {
+          "Create or Update CodeVersion": {
+            "$ref": "./examples/createOrUpdateCodeVersion.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CodeVersionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CodeVersionVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Code entity to create or update.",
+            "schema": {
+              "$ref": "#/definitions/CodeVersionResourceRequest"
+            }
+          }
```

Comment:
Why is a codeversion API returning a codecontainer resource?

Reply:
whoops, it shouldn't :D fixed